ThreeBotPackages/tfgrid/threebot_provisioning/actors/provisioning.py

```diff
@@ -1,9 +1,12 @@
 from Jumpscale import j
-
+import netaddr
+import ipaddress
 
 class provisioning(j.baseclasses.threebot_actor):
     def _init(self, **kwargs):
         self.token = j.clients.digitalocean.get("provisioning").token_
+        j.clients.threebot.explorer_addr_set("explorer.testnet.grid.tf")
```

Comment:
I do not think we should hardcode the testnet explorer address here. How would we use it for production when we launch the production network ?